lib/saluki-components/src/destinations/datadog/common/retry.rs

```diff
@@ -90,6 +130,24 @@ impl RetryConfiguration {
         self.retry_queue_max_size_bytes
     }
 
+    /// Returns the maximum size of the retry queue on disk, in bytes.
+    #[allow(unused)]
+    pub fn storage_max_size_bytes(&self) -> u64 {
+        self.storage_max_size_bytes
+    }
+
+    /// Returns the path to the directory where the retry queue will be stored on disk.
+    #[allow(unused)]
+    pub fn storage_path(&self) -> &str {
+        &self.storage_path
+    }
+
+    /// Returns the maximum disk usage ratio for storing transactions on disk.
+    #[allow(unused)]
```

Comment:
Looks like we don't actually have `with_disk_persistence` in use except for in tests (?) These unused functions would be used in combination with `with_disk_persistence` [here](https://github.com/DataDog/saluki/blob/9d02818eee138de7feb1f4a84b74a33cb9b3742f/lib/saluki-components/src/destinations/datadog/common/io.rs#L257)